cloud/ovirt/ovirt_networks_facts.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2016 Red Hat, Inc.
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+try:
+    import ovirtsdk4 as sdk
+except ImportError:
+    pass
+
+from ansible.module_utils.ovirt import *
+
+
+DOCUMENTATION = '''
+---
+module: ovirt_networks_facts
+short_description: Retrieve facts about one or more oVirt networks
+version_added: "2.3"
+description:
+    - "Retrieve facts about one or more oVirt networks."
+notes:
+    - "This module creates a new top-level C(ovirt_networks) fact, which
+       contains a list of networks."
+requirements:
+    - python >= 2.7
+    - ovirt-engine-sdk-python >= 4.0.0
+options:
+    pattern:
+      description:
+        - "Search term which is accepted by oVirt search backend."
+        - "For example to search network starting with string vlan1 use: name=vlan1*"
+extends_documentation_fragment: ovirt
+'''
+
+
+EXAMPLES = '''
+# Examples don't contain auth parameter for simplicity,
+# look at ovirt_auth module to see how to reuse authentication:
+
+# Gather facts about all networks named C<vlan1*>:
```

Comment:
I'd rephrase to 'Gather facts about all networks which names start with C<vlan1>: